packages/components/src/higher-order/navigate-regions/style.scss

```diff
@@ -1,6 +1,20 @@
 // Allow the position to be easily overridden to e.g. fixed.
 [role="region"] {
 	position: relative;
+
+	// Handles the focus when we programatically send focus to this region
+	&.interface-interface-skeleton__content:focus-visible::after {
+		position: absolute;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
+		content: "";
+		pointer-events: none;
+		outline: 4px solid $components-color-accent;
+		outline-offset: -4px;
+		z-index: z-index(".is-focusing-regions {region} :focus::after");
```

Comment:
I wonder if this should be extracted into a mixin or a `%` selector to be reused so we don't accidentally update one without the other.